dts/msm8974/msm8974-samsung-hlte-r07.dts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0x7E01FF01 7 0>, <0xB801FF01 7 0>, <0xB901FF01 7 0>;
+
+	hlte {
+		qcom,board-id = <1 0x0>;
+		qcom,msm-id = <126 0x10001>;
```

Comment:
```suggestion
```

this looks bogus and I really don't think any samsung dts would have this pair...

Reply:
That's what I thought. Looked too _generic_ to me, although my mind went "who knows?" at that time.
Those were based on lk2nd's log and my quick run-through of the documentation for board-id. However, I just can't find anything on board-id for non-Pro MSM8974.

Reply:
lk2nd has logged what it has read from the hardware fuses using the qcom's code that most oems also use. However samsung is one of the oems who doesn't use that code and instead has their own msm-id scheme with the long 4 byte value + rev number. If you want to make sure downstream works, you /do/ need to use those properties to override lk2nd's guess but you need to set the desired values from downstream dts, i.e. like here:

https://github.com/msm8916-mainline/lk2nd/blob/b02d7582ef456ba1e0297d8a2beb3e098d1d8ce9/dts/msm8916/msm8916-samsung-r08.dts#L48-L49

which is usually only necessary if the rev you need is different from the rev that samsung's bl picks in lk2nd dts, since lk2nd tries to find and reuse samsung's bootloader choice when it can. (not sure how robust is that though)